tests/lib/eslint/eslint.js

```diff
@@ -2937,6 +2967,117 @@ describe("ESLint", () => {
                     assert.strictEqual(results[0].messages[0].message, "Program is disallowed.");
                 });
 
+                // https://github.com/eslint/eslint/issues/18575
+                describe("on Windows", () => {
+                    if (os.platform() !== "win32") {
+                        return;
+                    }
+
+                    let otherDriveLetter;
+                    const exec = util.promisify(require("node:child_process").exec);
+
+                    before(async () => {
```

Comment:
Can you add a comment here explaining what's happening? I'm not quite sure I get it now, and I'm sure in another year I'll have no idea. :smile:

Reply:
I added comments in 31938bed6b11c61b18cbea9030c36da1fc8fa230. Hope it's clearer now.